addons/mail/static/src/dropzone/dropzone.js

```diff
@@ -0,0 +1,54 @@
+/** @odoo-module **/
+
+import { Component, onWillUnmount, useState } from "@odoo/owl";
+
+export class Dropzone extends Component {
+    static props = {};
+    static template = "mail.dropzone";
+
+    setup() {
+        const preventDefault = ev => ev.preventDefault;
```

Comment:
note that you can use the `useExternalListener` hook here:
```js
useExternalListener(window, "dragover", ev => ev.preventDefault);
```
and you don't need to unregister it yourself, the hook will take care of it

Reply:
That's good to know, thanks 🙂